calculator/Presenter/CalculatorPresenter.swift

```diff
@@ -11,11 +11,17 @@ protocol CalculatorDelegate: AnyObject {
     func showOnCalculatorDisplay(_ content: String)
 }
 
-class CalculatorPresenter {
-    weak var delegate: CalculatorDelegate?
+class CalculatorPresenter: PresenterProtocol {
+    var router: RouterProtocol?
+    var interactor: InteractorProtocol?
+    var view: ViewProtocol?
```

Comment:
Challenge 😄
I think this reference must be weak
https://github.com/eduardo-glb/MyPersonalCV/blob/master/img/VIPER.png
what do you think?